src/libs/Browser/index.web.js

```diff
@@ -50,8 +51,42 @@ function isMobileSafari() {
     return /iP(ad|od|hone)/i.test(userAgent) && /WebKit/i.test(userAgent) && !(/(CriOS|FxiOS|OPiOS|mercury)/i.test(userAgent));
 }
 
+function openRouteInDesktopApp(shortLivedAuthToken = '', email = '') {
+    const params = new URLSearchParams();
```

Comment:
Can you please add a code comment here explaining _why_ the params have to be passed this way?

Reply:
Oh, sorry... I am just thinking of a comment like this:

> The session information needs to be passed to the Desktop app, and the only way to do that is by using query params. There is no other way to transfer the data.